Makefile

```diff
@@ -20,29 +20,26 @@ $(TARGETS):
 # Project-specific targets go here
 deploy: nettest
 
-nettest:
-	$(SCRIPTS_DIR)/build_image.sh -i nettest -f package/Dockerfile.nettest
+nettest: package/.image.nettest
 
 e2e: vendor/modules.txt clusters
 
 else
 
 # Not running in Dapper
 
+include Makefile.images
+
 # Shipyard-specific starts
 clusters deploy e2e nettest post-mortem release unit-test validate: dapper-image
 
 dapper-image: export SCRIPTS_DIR=./scripts/shared
 
-dapper-image:
-	$(SCRIPTS_DIR)/build_image.sh -i shipyard-dapper-base -f package/Dockerfile.dapper-base $(dapper_image_flags)
+dapper-image: package/.image.shipyard-dapper-base
 
 .DEFAULT_GOAL := validate
 # Shipyard-specific ends
 
 include Makefile.dapper
 
 endif
-
-# Disable rebuilding Makefile
```

Comment:
Why remove this?

Reply:
> The `MAKECMDFLAGS` change could do with an explanation in the commit message (or even a separate commit!).

Sure, I'll add a comment.

It had to be in this commit or it won't work (probably before it would work but I don't see a good reason to split it out now)